README.md

```diff
@@ -85,6 +94,29 @@ Your output could be something like:
 The most common mood is :-)
 ```
 
-**think**: Should you use an array or a hash to solve this problem? Why?
-
 **think**: What if we eventually want to add feelings to our analysis? Can we write this code in a way that will prevent us from having to re-write it later?
+**think**: Should you use an array or a hash to solve this problem? Why?
+A: The analyze_mood method should be re-written so that we could loop through all the feelings, and we would have to have a counter inside the data structure to keep track of how many words matched in the entry.
+A: I think a hash with values as arrays would work:
+
+FEELINGS = {
+  happy: [%w(yay good great), 0, ":-)"],
+  sad: [%w(terrible awful horrible), 0, ":-("]
+}
```

Comment:
Interesting. I don't know how I feel about this combining of many things of different needs in one big array. I take that back, I do know how I feel, I don't love it. It works, but I don't think it's an intuitive solution. I like grouping all the things, but probably not using an array. This would make me happier:

```
FEELINGS = {
  happy: {words: %w(yay good great), count:0, emoticon: ":-)"}
  #...
```

But I still don't really like the count being mixed up in there. I'd rather that be a separate data structure/variable.